src/led.h

```diff
@@ -15,4 +14,74 @@
 #define LED_h   0x12
 #define LED_dp  0x13
 
-void filldisplay(uint8_t pos, uint8_t val, __bit dp);
+const uint8_t __at (0x1000) ledtable[]
```

Comment:
I still don't know if **code** can be run in the eeprom zone.
For sure the cpu can access this zone using regular pointer in the range starting at 0x1000.
The difference that I see, is that this zone can be programmed from the application itself (via the specific IAP registers).
So for the moment, you cannot say that you can use 5k code - I only found interesting to put static data in that zone (quite efficient, since you can use dph=0x10, and then dpl from 0 to 255 to access up to 256 bytes from the ledtable[], then dph=0x11 for the pre-shifted ledtable2 for digit3).